Documentation/project-docs/copyright.md

```diff
@@ -0,0 +1,34 @@
+Copyright
+=========
+
+The .NET Core copyright is held by ".NET Foundation and Contributors". 
+
+The [.NET Foundation](http://www.dotnetfoundation.org/) is an independent organization that encourages open development and collaboration around the .NET ecosystem.
```

Comment:
I may be pedantic, but "independent" of what? How is it different from "The .NET Foundation is an organization that..."